test/test_serialization.h

```diff
@@ -1006,7 +1008,7 @@ TEST(serialization, graph_model_and_weights) {
   fully_connected_layer f2(2, 2);
   softmax_layer a2(2);
   fully_connected_layer f3(2, 2);
-  elu_layer a3(2);
+  elu_layer a3((size_t)2);
```

Comment:
for convention do:
`elu_layer a3(size_t(2))` or `elu_layer a3(static_cast<size_t>(2))`